content/blog/create-a-cold-tiered-volume-and-use-hpe-ezmeral-data-fabric-object-store-as-remote-target.md

```diff
@@ -0,0 +1,656 @@
+An Alias also contains the Access Key and Secret Key used by your administrator or IAM User.
+
+##### 1. You first use the "[mc alias list](https://docs.datafabric.hpe.com/72/ReferenceGuide/mc-alias-list.html)" command to view the default Alias in the following systems.
+
+❗Note: If you are using self-signed TLS certificates or installed the cluster by Installer, you have to copy <ins>/opt/mapr/conf/ca/chain-ca.pem</ins> to <ins>~/.mc/certs/CAs/</ins> on the node running `mc`.
+The meaning of this step is the same as you imported the self-issued CA to the keytool of the JVM earlier, `mc` also needs to import the self-issued CA to communicate with the S3server of the Object Store.
+
+```shell
+sudo -E -u mapr /opt/mapr/bin/mc alias list
+```
+
+Sample output:
+
+```
+gcs
+  URL       : https://storage.googleapis.com
+  AccessKey : YOUR-ACCESS-KEY-HERE
+  SecretKey : YOUR-SECRET-KEY-HERE
+  API       : S3v2
+  Path      : dns
+
+local
+  URL       : http://localhost:9000
+  AccessKey :
+  SecretKey :
+  API       :
+  Path      : auto
+
+play
+  URL       : https://play.min.io
+  AccessKey : XXXXXXXXXXXXXXXXXXXX
+  SecretKey : XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+  API       : S3v4
+  Path      : auto
+
+s3
+  URL       : https://s3.amazonaws.com
+  AccessKey : YOUR-ACCESS-KEY-HERE
+  SecretKey : YOUR-SECRET-KEY-HERE
+  API       : S3v4
+  Path      : dns
+```
+
+##### 2. Generate S3 keys to authenticate your administrator
+
+The cluster administrator (typically the mapr user) must authenticate to the Object Store cluster and generate S3 keys (accessKey and secretKey) on the default Object Store account.
+Perform this operation before performing any CLI operations in Object Store.
+
+If the cluster is secure, use [maprlogin](https://docs.datafabric.hpe.com/72/SecurityGuide/ThemaprloginUtility.html) to authenticate the cluster administrator, and then generate the keys:
+
+```shell
+maprcli s3keys generate -domainname primary -accountname default -username mapr -json
+```
+
+🗒Note: An Object Store cluster has a domain, accounts, buckets, users, and access policies associated with it.
+Installing Object Store in a cluster provides a primary domain and a default account.
+
+Sample output:
+
+```json
+{
+  "timestamp":1676472096994,
+  "timeofday":"2023-02-15 10:41:36.994 GMT+0800 PM",
+  "status":"OK",
+  "total":1,
+  "data":[
+          {
+            "accesskey":"XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+            "secretkey":"XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+          }
+  ]
+}
+```
+
+🗒Note: If you encounter any problem when generating the S3 keys, refer to this page: [Generate S3 Keys to Authenticate Users and Applications](https://docs.datafabric.hpe.com/72/MapROverview/object-store-get-started.html#object-store-get-started__section_x4h_w13_4tb).
+
+##### 3. Use the `mc alias set` command to create an alias for admin user

```

Comment:
Again, make sure you take a look at this in preview mode to ensure you have the right level of bulletization going on.